src/shared/ui/Input/Input.module.scss

```
@@ -0,0 +1,31 @@
+.input {
+  outline: none;
+  width: 350px;
+  height: 84px;
+  font: var(--font-line-m);
+  padding-left: 28px;
+  color: #afadb5;
+  background: #ffffff;
+  border: none;
+  box-shadow: 0px 4px 120px rgba(175, 173, 181, 0.15);
+  &:focus {
+    background: #ffffff;
+    border: 1px solid #518581;
+    color: #151411;
+  }
+}
+
+.input_disabled:disabled {
+  width: 299.72px;
+  height: 84px;
+  padding: 26px auto 26px 28px;
+  background: #ffffff;
+  box-shadow: 0px 4px 120px rgba(175, 173, 181, 0.15);
+  font-family: "Roboto";
```

здесь и где ещё встретится: нужно дополнительно указывать fallback-шрифт, чтобы, если не подгрузился основной, был использован запасной:
```suggestion
  font-family: "Roboto", sans-serif;
```